bundles/framework/layerlist/view/LayerViewTabs/LayerList/LayerCollapse/LayerCollapsePanel.jsx

```diff
@@ -157,7 +170,7 @@ const LayerCollapsePanel = (props) => {
             className={`t_group gid_${group.getId()}`}
             // data-attr doesn't seem to work for the panel in AntD-version 4.8.5
             data-gid={group.getId()}
-            header={group.getTitle()}
+            header={typeof group.getTitle() === 'object' ? '' : group.getTitle()}
```

Comment:
The title is an object for groups generated at runtime? Like statistical maps/my places etc? We should change the title on them instead.